app/components/Chatbot/widgit.css

```diff
@@ -20,3 +20,29 @@
     width: 100%;
   }
 }
+.height-70 {
+  height: 70vh;
+  overflow-x: hidden;
+}
+.widget-ask {
+  position: sticky;
+  bottom: 0;
+}
+.pointer {
+  position: absolute;
+  right: 0;
+  margin: 4px;
+}
+.right-icon {
+  padding-right: var(--spacing-56);
+}
+.warning-floating {
+  position: fixed;
+  right: 7vw;
+  z-index: 100;
+  bottom: 5vw;
+  width: 11vw;
+}
+.red {
+  color: #d40000;
```

Comment:
We'll want to define this and reference it as a variable, plus put this class in the same place as the other color classes in root.css